src/uu/cut/src/cut.rs

```diff
@@ -449,9 +556,13 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
                 };
 
                 let only_delimited = matches.get_flag(options::ONLY_DELIMITED);
+                let whitespace_delimited = matches.get_flag(options::WHITESPACE_DELIMITED);
```

Comment:
I think we can make a bit of an improvement here by making the delimiter a new `enum`:
```rust
enum Delimiter {
    Whitespace,
    String(String),
    None,
}
```

Reply:
done with two options:
```rust
enum Delimiter {
    Whitespace,
    String(String),
}
```